src/main/scala/codacy/jshint/package.scala

```diff
@@ -14,12 +14,15 @@ package object jshint {
     es3 -> false,
     forin -> false,
     freeze -> false,
+    futurehostile -> false,
```

Comment:
What is the meaning of this object on the big scope of this repo? Why only 3 of the 6 are added here?

Reply:
The other 3 were already there but were not added to the `patterns.json`

Reply:
They are the base settings on top of which we apply our settings.. It is not in 1:1 relationship with patterns.json since jshint seeings don't reflect one setting is one pattern.. So they were set up separately.
JSHint works in a way very different from how Codacy works, that's why there was some manual work to make work _similarly_

Reply:
Thanks for the explanation, LGTM!